README.md

```diff
@@ -1,131 +1,2469 @@
+ - [WritablePowerPortRequest](docs/WritablePowerPortRequest.md)
+ - [WritablePowerPortTemplateRequest](docs/WritablePowerPortTemplateRequest.md)
+ - [WritablePrefixRequest](docs/WritablePrefixRequest.md)
+ - [WritableRackRequest](docs/WritableRackRequest.md)
+ - [WritableRearPortRequest](docs/WritableRearPortRequest.md)
+ - [WritableRearPortTemplateRequest](docs/WritableRearPortTemplateRequest.md)
+ - [WritableServiceRequest](docs/WritableServiceRequest.md)
+ - [WritableVMInterfaceRequest](docs/WritableVMInterfaceRequest.md)
 
-```go
-package main
 
-import (
-	"context"
-	"fmt"
+## Documentation For Authorization
 
-	nb "github.com/nautobot/go-nautobot/pkg/nautobot"
-)
 
-func check(err error) {
-	if err != nil {
-		panic(err)
-	}
-}
+Authentication schemes defined for the API:
+### basicAuth
 
-func main() {
-	token, err := nb.NewSecurityProviderNautobotToken("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
-	check(err)
+- **Type**: HTTP basic authentication
 
-	c, err := nb.NewClientWithResponses(
-		"https://demo.nautobot.com/api/",
-		nb.WithRequestEditorFn(token.Intercept),
-	)
-	check(err)
+Example
 
-	ctx := context.Background()
+```go
+auth := context.WithValue(context.Background(), nautobot.ContextBasicAuth, nautobot.BasicAuth{
+	UserName: "username",
+	Password: "password",
+})
+r, err := client.Service.Operation(auth, args)
+```
 
-	resp, err := c.DcimManufacturersListWithResponse(ctx, &nb.DcimManufacturersListParams{})
-	check(err)
+### cookieAuth
 
-	fmt.Printf("%v", string(resp.Body))
+- **Type**: API key
+- **API key parameter name**: sessionid
+- **Location**: 
 
-}
+Note, each API key must be added to a map of `map[string]APIKey` where the key is: sessionid and passed in as the auth context for each request.
 
-```
+Example
 
-### Other usage examples
+```go
+auth := context.WithValue(
+		context.Background(),
+		nautobot.ContextAPIKeys,
+		map[string]nautobot.APIKey{
+			"sessionid": {Key: "API_KEY_STRING"},
+		},
+	)
+r, err := client.Service.Operation(auth, args)
+```
 
-- [Network Automation with Go Book](https://github.com/PacktPublishing/Network-Automation-with-Go/blob/main/ch06/nautobot/main.go)
+### tokenAuth
 
-## Local Development
+- **Type**: API key
+- **API key parameter name**: Authorization
+- **Location**: HTTP header
 
-> This repository uses Python ìnvoke`package to run development tasks. You can install it with`pip`.
+Note, each API key must be added to a map of `map[string]APIKey` where the key is: Authorization and passed in as the auth context for each request.
 
-### Run tests locally
+Example
 
-```
-invoke tests
+```go
+auth := context.WithValue(
+		context.Background(),
+		nautobot.ContextAPIKeys,
+		map[string]nautobot.APIKey{
+			"Authorization": {Key: "API_KEY_STRING"},
+		},
+	)
+r, err := client.Service.Operation(auth, args)
 ```
 
-> Hint: If you get a build fail during testing, check that you are not limiting container memory to 2GB. Upgrade to 4GB.
 
-### Trigger the release manually
```

Comment:
please, keep the memory warning if you als0 need it

and also keep the small section about how to run the workflow in GitHub actions to trigger the release

Reply:
Done